pkg/cmd/roachprod-microbench/main.go

```diff
@@ -166,6 +177,8 @@ func makeCompareCommand() *cobra.Command {
 	cmd.Flags().StringVar(&config.slackToken, "slack-token", config.slackToken, "pass a slack token to post the results to a slack channel")
 	cmd.Flags().StringVar(&config.slackUser, "slack-user", config.slackUser, "slack user to post the results as")
 	cmd.Flags().StringVar(&config.slackChannel, "slack-channel", config.slackChannel, "slack channel to post the results to")
+	cmd.Flags().Float64Var(&config.threshold, "threshold", config.threshold, "threshold for detecting perf regression")
```

Comment:
Nit: It's not clear from the description alone how this `threshold` is expressed. From the implementation, it appears to be a percentage (difference) normalized to (0, 1]. Why not use a percentage instead?

Reply:
Not any strong preference for any approach. I wanted to use absolute difference but now I feel percentage is better since every other comparison has percentage.